[2024-09-11T11:07:22.019+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-11T11:07:22.076+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: max_min_dag.generate_random_numbers scheduled__2024-09-10T00:00:00+00:00 [queued]>
[2024-09-11T11:07:22.099+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: max_min_dag.generate_random_numbers scheduled__2024-09-10T00:00:00+00:00 [queued]>
[2024-09-11T11:07:22.100+0000] {taskinstance.py:2865} INFO - Starting attempt 3 of 4
[2024-09-11T11:07:22.142+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_random_numbers> on 2024-09-10 00:00:00+00:00
[2024-09-11T11:07:22.166+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=67) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-11T11:07:22.183+0000] {standard_task_runner.py:72} INFO - Started process 68 to run task
[2024-09-11T11:07:22.182+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'max_min_dag', 'generate_random_numbers', 'scheduled__2024-09-10T00:00:00+00:00', '--job-id', '255', '--raw', '--subdir', 'DAGS_FOLDER/simple_dag.py', '--cfg-path', '/tmp/tmpyc2m37fz']
[2024-09-11T11:07:22.187+0000] {standard_task_runner.py:105} INFO - Job 255: Subtask generate_random_numbers
[2024-09-11T11:07:22.309+0000] {task_command.py:467} INFO - Running <TaskInstance: max_min_dag.generate_random_numbers scheduled__2024-09-10T00:00:00+00:00 [running]> on host 0a0201be4888
[2024-09-11T11:07:22.500+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Denis' AIRFLOW_CTX_DAG_ID='max_min_dag' AIRFLOW_CTX_TASK_ID='generate_random_numbers' AIRFLOW_CTX_EXECUTION_DATE='2024-09-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-10T00:00:00+00:00'
[2024-09-11T11:07:22.503+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-11T11:07:22.596+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-11T11:07:22.621+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-11T11:07:22.623+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=max_min_dag, task_id=generate_random_numbers, run_id=scheduled__2024-09-10T00:00:00+00:00, execution_date=20240910T000000, start_date=20240911T110722, end_date=20240911T110722
[2024-09-11T11:07:22.690+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-11T11:07:22.753+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-11T11:07:22.757+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
